js/background.js

```diff
@@ -1,80 +1,76 @@
 /* This script runs in the background from when the app is loaded */
 
-var Domain = "https://channeli.in";
-var Host = "channeli.in";
+var NET_CHECK_DOMAIN = "http://info.cern.ch/";
```

Comment:
This URL returns the fastest response I have ever seen as it is quite possibly the simplest page on the Internet. No CSS/JS, oldest possible HTML and a few lines of plain text only